modules/ROOT/nav.adoc

```diff
@@ -332,6 +333,7 @@
 ***** xref:tinydrive-pick.adoc[The Pick API]
 ***** xref:tinydrive-type-interfaces.adoc[TypeScript interfaces]
 **** xref:tinydrive-changelog.adoc[Changelog]
+*** xref:tableofcontents.adoc[Table of Contents]
```

Comment:
This needs moving so that it's in alphabetical order.

- [ ] Move to be before the `Tiny Drive` plugin docs